### PR TITLE
feat: Integrate with universal-registrar to create DID's

### LINKED ISSUE #195: Integrate with universal-registrar to create DID's
Integrate with universal-registrar in vc-rest

### PR DESCRIPTION
**Pull request by fqutishat**
closes #195

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/universal-registrar/docker-compose.yml [hunk 1]
```diff
@@ -4,28 +4,25 @@
...
 #
 
-version: "3.5"
-networks:
-  default:
-    name: universal-registrar
+version: '3'
+
 services:
-  driver-did-sov:
```
Comment (troyronda) [inline test/bdd/fixtures/universal-registrar/docker-compose.yml:12]: how do we handle sov?

File: test/bdd/fixtures/universal-registrar/docker-compose.yml [hunk 2]
```diff
@@ -4,28 +4,25 @@
...
-      uniregistrar_driver_did_sov_poolConfigs: ${UNIREGISTRAR_DRIVER_DID_SOV_POOLCONFIGS}
-      uniregistrar_driver_did_sov_poolVersions: ${UNIREGISTRAR_DRIVER_DID_SOV_POOLVERSIONS}
-      uniregistrar_driver_did_sov_walletName: ${UNIREGISTRAR_DRIVER_DID_SOV_WALLETNAME}
-      uniregistrar_driver_did_sov_trustAnchorSeed: ${UNIREGISTRAR_DRIVER_DID_SOV_TRUSTANCHORSEED}
-    ports:
-      - "9082:9080"
   driver-did-v1:
     image: universalregistrar/driver-did-v1:latest
-    environment:
-      uniregistrar_driver_did_v1_trustAnchorSeed: ${UNIREGISTRAR_DRIVER_DID_V1_TRUSTANCHORSEED}
```
Comment (troyronda) [inline test/bdd/fixtures/universal-registrar/docker-compose.yml:25]: the seed isn’t needed?

File: test/bdd/fixtures/universal-registrar/docker-compose.yml (hunk 1 above)
Comment (fqutishat) [reply test/bdd/fixtures/universal-registrar/docker-compose.yml:12]: there is issue in sov driver with local uni-registrar

Comment (fqutishat) [reply test/bdd/fixtures/universal-registrar/docker-compose.yml:12]: @sudeshrshetty ^^^^

Comment (fqutishat) [reply test/bdd/fixtures/universal-registrar/docker-compose.yml:12]: @troyronda @sudeshrshetty https://github.com/trustbloc/edge-service/pull/207/files#diff-a2c2abd8f0260e030bfc497f1d7f166fL23

File: test/bdd/fixtures/universal-registrar/docker-compose.yml (hunk 2 above)
Comment (fqutishat) [reply test/bdd/fixtures/universal-registrar/docker-compose.yml:25]: not needed for driver-did-v1 https://github.com/decentralized-identity/universal-registrar/blob/3c26c4c1abc338ff502ea8eedeedd4538e713836/.env#L18

File: test/bdd/fixtures/universal-registrar/docker-compose.yml (hunk 1 above)
Comment (fqutishat) [reply test/bdd/fixtures/universal-registrar/docker-compose.yml:12]: @troyronda @sudeshrshetty we handle sov using https://uniregistrar.io/ not local

**Pull request merged**